### PR TITLE
#137599493 user authentication

### PR DESCRIPTION
**Pull request by cyrielo**
#### What does this PR do?
This PR solves authentication issues in the frontend to work with the new Andela Authentication mechanism
#### Description of Task to be completed?
As a targeted user of Zhishi, I can log into the application in order to GTD.
#### How should this be manually tested?
By logging in into the application
#### What are the relevant pivotal tracker stories?
#137599493
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: app/js/auth/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import cookie from 'js-cookie';
 
 class Auth {
   andelaLoginUrl() {
-    return `http://authentication.andela.com/auth/google?redirect_url=http://${
+    return `http://api-staging.andela.com/login?redirect_url=http://${
```
Comment (andela-oakinniranye) [inline app/js/auth/index.js]: Find a way to fetch this with an `env` variable!

File: tools/distServer.js [hunk 2]
```diff
@@ -30,11 +30,6 @@ if (!isDeveloping) {
 
 app.use(cookieParser());
 app.use((req, res, next) => {
```
Comment (andela-oakinniranye) [inline tools/distServer.js]: 👊

File: app/js/config/CookieVariables.js [hunk 3]
```diff
@@ -6,5 +6,6 @@ module.exports = {
   user_logged_in: "iwengjk",
   referrer: 'ehdieleda',
   first_time_marker: 'yijeusheuw',
-  apiUrl: 'kdjndnvjds'
+  apiUrl: 'kdjndnvjds',
+  jwt : 'jwt-token'
```
Comment (andela-oakinniranye) [inline app/js/config/CookieVariables.js]: please confirm the values of these variables and whether they can be more useful/readable

**Comment by andela-oakinniranye** [comment]
Looking good, couple of comments and `ci`!

File: app/js/config/CookieVariables.js (hunk 3 above)
Comment (cyrielo) [inline app/js/config/CookieVariables.js]: The values are used as keys to store cookies. I believe the values were intentionally made cryptic so cookie viewers will not guess at first glance what the cookie is about. Aside from the jwt-token.

**Comment by andela-bmakinwa** [comment]
@cyrielo nice work. Kindly refactor your commit messages to follow the convention [here](https://github.com/andela/engineering-playbook/wiki/Conventions#commit-message)

**Pull request merged**